common/scripts/build-fwts-arm.sh

```diff
@@ -0,0 +1,151 @@
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# Redistributions of source code must retain the above copyright notice, this
+# list of conditions and the following disclaimer.
+#
+# Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# Neither the name of ARM nor the names of its contributors may be used
+# to endorse or promote products derived from this software without specific
+# prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+#
+# This script uses the following environment variables from the variant
+#
+# VARIANT - build variant name
+# TOP_DIR - workspace root directory
+# CROSS_COMPILE - PATH to GCC including CROSS-COMPILE prefix
+# PARALLELISM - number of cores to build across
+
+TOP_DIR=`pwd`
+FWTS_PATH=fwts
+FWTS_BINARY=fwts_output
+RAMDISK_PATH=ramdisk
+FWTS_DEP=$RAMDISK_PATH/fwts_build_dep
+
+. $TOP_DIR/../../common/config/common_config.cfg
+. $TOP_DIR/../../common/scripts/cross_toolchain-arm.sh
+
+BUILD_PLAT=$1
+BUILD_TYPE=$2
+
+if ! [[ $BUILD_TYPE = S ]] && ! [[  $BUILD_TYPE = F  ]] ; then
+    echo "Please provide a Build type."
+    echo "Usage build-fwts.sh <target> <S/F>"
+    echo "S->Standalone BBR,F->Full systemready"
+    exit
+fi
+
+if [[ $BUILD_TYPE = S ]]; then
+    BBR_DIR=$TOP_DIR/../..
+else
+    BBR_DIR=$TOP_DIR/bbr-acs
+fi
+
+init()
+{
+    if [[ $BUILD_TYPE = S ]]; then
+        mkdir -p $TOP_DIR/$RAMDISK_PATH
+    fi
+    cp -r $BBR_DIR/common/fwts_build_dep $RAMDISK_PATH
+}
+
+do_build()
+{
+    pushd $TOP_DIR/$FWTS_PATH
+    DEF_PATH=$PATH
+    PATH=$(getconf PATH) #Reset path to avoid cross compiler mismatch
+    PATH="$PATH:$CROSS_COMPILE_DIR"
+
+    echo $BBR_DIR
+
+    if [ "$BUILD_PLAT" = "IR" ]; then
+        if ! patch -R -s -f --dry-run -p1 < $BBR_DIR/ebbr/config/IR_VER.patch ; then
+        echo "Applying FWTS Patch ..."
+        patch -p1 < $BBR_DIR/ebbr/config/IR_VER.patch
+        fi
+    fi
+    if [ "$BUILD_PLAT" = "ES" ]; then
```

Comment:
The SystemReady ES band accepts only 64b systems for now so I think you can drop support for ES+32b.

Reply:
Ok, i'll remove instruction related to ES.

Reply:
addressed

Reply:
addressed in [`[review]` fixup commit](https://github.com/ARM-software/bbr-acs/pull/11/commits/1e150ed066f2c2bb0693721d0cada03c71abd8cb#diff-cb7f13fe0d41d1fef7525b4f7ae1c8a671d25d9df797093c76005cfdf1d22a0cR51).